lib/msal-common/src/utils/TimeUtils.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @hidden
+ */
+export class TimeUtils {
+    /**
+     * Returns time in seconds for expiration based on string value passed in.
+     *
+     * @param expiresIn
+     */
+    static parseExpiresIn(expiresIn: number): number {
+        // if AAD did not send "expires_in" property, use default expiration of 3599 seconds, for some reason AAD sends 3599 as "expires_in" value instead of 3600
+        if (!expiresIn) {
+            expiresIn = 3599;
+        }
+        return expiresIn;
+    }
+
+    /**
+     * return the current time in Unix time. Date.getTime() returns in milliseconds.
+     */
+    static now(): number {
+        return Math.round(new Date().getTime() / 1000.0);
```

Comment:
`getTime()` returns ms already, shouldn't need to do any additional math.

Reply:
Also, can the function be named `nowMs` to indicate you are getting ms?

Reply:
Ah, misread the comments...another reason why labelling the units here makes it clear we are working with seconds.

Reply:
I changed the comments in the silent flow PR, it threw me off as well.